fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/ContentDigest.java

```diff
@@ -38,20 +41,39 @@
 
     private static final Logger LOGGER = getLogger(ContentDigest.class);
 
+    private final static String DEFAULT_DIGEST_ALGORITHM_PROPERTY = "fcrepo.persistence.defaultDigestAlgorithm";
```

Comment:
Thoughts on refactoring: "fedoraconfig:defaultDigestAlgorithm"?
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraTypes.java#L90

Reply:
Based on your slack response from the other day, I was thinking we would end up pulling the `fedoraconfig:defaultDigestAlgorithm` since it seems unnecessary with the removal of fcr:fixity and emphasis on the Want-Digest header. I can do so now, and update the documentation if you'd like.

On the other hand, it might make sense to reimagine it as a way for clients to tell fedora what digest algorithm to use for content addressing in a particular ocfl object, allowing them to override the system wide `DEFAULT_DIGEST_ALGORITHM`. But that likely doesn't fit into this PR, since it would probably come with some twists in terms of allowing clients to change existing objects. From what peter was saying I had the impression that was an allowed ocfl operation, but I would expect it would require significant changes to the object.

Reply:
Thanks. Let's take a wait-and-see on `fedoraconfig:defaultDigestAlgorithm`.